### ISSUE TITLE
Fails if root gradle project directory is not also root git directory

### ISSUE THREAD

**Issue by gpothier** [ReactiveCircus/app-versioning#12]
The plugin fails with this error if the Gradle root directory is a subdirectory of the git repository: "Android App Versioning Gradle Plugin works with git tags but root project 'android' is not a valid git repository"

For instance, in a Ionic app, the Gradle project is within an `android` directory inside the repository. From the command line, `git describe --tags` works fine from within any subdirectory of the repository. I don't know if GitClient works the same, but the plugin checks if gitRefsDirectory exists before initializing GitClient.

**Comment by ychescale9** [comment]
Thanks for reporting this. I just released `0.8.1` which allows you to specify the git root directory explicitly:

```kotlin
appVersioning {
    gitRootDirectory.set(rootProject.file("../"))
}
```

Let me know if it works :).

**Comment by gpothier** [comment]
Wow thanks a lot, it works!
